scipy/stats/_stats_py.py

```diff
@@ -10189,7 +10189,8 @@ def quantile_test(x, *, q=0, p=0.5, alternative='two-sided'):
 
 def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     r"""
-    Compute the Wasserstein-1 distance between two discrete distributions.
+    Compute the Wasserstein-1 distance between two discrete distributions
+    with respect to the Euclidean norm.
```

Comment:
This should be a [one-line summary](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary).